.github/workflows/documentation.yml

```diff
@@ -57,7 +57,8 @@ jobs:
           git config --global user.email "actions@github.com"
           git config --global user.name "gh-actions"
           yarn add nodejieba
-          yarn install --frozen-lockfile
+          yarn install --frozen-lockfile --production=false
```

Comment:
```suggestion
          yarn install --frozen-lockfile --production=true
```

Should we mark the production to be true?

Reply:
This should not be `true`, otherwise it will not install devDependencies (thus will not build anything). This parameter is added to tell `yarn` to install devDependencies, preventing accidental `NODE_ENV=production` causing build issues.

https://classic.yarnpkg.com/lang/en/docs/cli/install/
<img width="822" alt="image" src="https://user-images.githubusercontent.com/55270174/174950924-304b9413-6b07-4919-ab5d-72e5f1adcb89.png">

Reply:
Of course, there is no devDependencies in this repo lol (which `--production` makes no difference). This is just some practice commonly used in CIs.